sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/SignatureHashResolver.java

```diff
@@ -16,14 +16,15 @@ class SignatureHashResolver {
         DEFAULT.put(SignatureAlgorithm.ES256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.ES256K, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.ES384, HashAlgorithm.SHA_384);
-
         DEFAULT.put(SignatureAlgorithm.ES512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.RS256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.RS384, HashAlgorithm.SHA_384);
         DEFAULT.put(SignatureAlgorithm.RS512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.PS256, HashAlgorithm.SHA_256);
         DEFAULT.put(SignatureAlgorithm.PS512, HashAlgorithm.SHA_512);
         DEFAULT.put(SignatureAlgorithm.PS384, HashAlgorithm.SHA_384);
+        // The specification says we should use SHA_512 but the service only supports SHA-256 for now for EdDSA.
```

Comment:
Do you want to add a TODO: to add a follow-up when the service does support SHA256?